earth_enterprise/src/server/geselectpublishroot

```diff
@@ -15,12 +15,31 @@
 # limitations under the License.
 #
 
+use strict;
+use Getopt::Long qw(:config no_ignore_case no_auto_abbrev);
+use FindBin qw($Script);
 
-# make sure we're running as root
-die "You must run as root\n" unless $> == 0;
+my $cmd_args_good = 0;
+my $help = 0;
+my $no_prompt = 1;
+
+sub usage {
+  warn "usage: $Script [-h] <publish_root_dir>\n";
+  warn "   Update the GE http server to point to the given\n";
```

Comment:
Let's document this bug in [release notes](https://github.com/google/earthenterprise/blob/master/docs/geedocs/5.3.0/answer/7160007.html)  in section `Resolved Issues`